modules/system-management/pages/memory-management.adoc

```diff
@@ -79,7 +79,7 @@ There are three memory states:
 Healthy:: There is a healthy amount of free memory.
 All system operations run normally for best performance.
 Alert:: There is a shortage of free memory.
-TigerGraph's Graph Processing Engine (GPE) starts to slow down processing new requests to allow long-running queries to finish and release memory.
+TigerGraph's GPE starts to slow down processing new requests to allow long-running queries to finish and release memory.
```

Comment:
We should spell out the full term when first mentioning it on a page. I think you can go back to the top of the page and spell out GPE.